apps/explorer/src/ui/SenderRecipient.tsx

```diff
@@ -0,0 +1,142 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { cva } from 'class-variance-authority';
+
+import { ReactComponent as DoneIcon } from '~/assets/SVGIcons/CheckFill.svg';
+import { ReactComponent as StartIcon } from '~/assets/SVGIcons/Start.svg';
+import { Heading } from '~/ui/Heading';
+import { Link } from '~/ui/Link';
+import { Text } from '~/ui/Text';
+
+const senderRecipientAddressStyles = cva(
+    ['break-all flex flex-row gap-2 w-full items-center'],
+    {
+        variants: {
+            isCoinTransfer: {
+                true: 'ml-6',
```

Comment:
This really shouldn't be on this component, because it's only relevant for the composing parent. We should keep components agnostic to the context they're, and make concerns like spacing are handled outside of the components (this really applies to both base components, and components in general).